charts/gardener-extension-registry-cache/templates/configmap-imagevector-overwrite.yaml

```diff
@@ -9,5 +9,5 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
 data:
   images_overwrite.yaml: |
-{{ toYaml .Values.imageVectorOverwrite | indent 4 }}
+{{ .Values.imageVectorOverwrite | indent 4 }}
```

Comment:
Find more details in https://github.com/gardener/gardener-extension-registry-cache/pull/80 why we did this change. Otherwise, `imageVectorOverwrite` is not human-readable.

I would rather suggest other components to adopt a change similar to https://github.com/gardener/gardener-extension-registry-cache/pull/80.

/hold

Reply:
Ok, got it! :) 
I need to adopt our tooling then, thank you!